### DISCUSSION TITLE
Uncaught (in promise): Error: "CapcitorMyPlugin" plugin is not implemented on ios CapacitorExceptio

### DISCUSSION THREAD

**Discussion by achakra21** [ionic-team/capacitor#4840] [General]
I have implemented custom plugin for capacitor i am getting below error when i am running the project where i am integrated the plugin

` Uncaught (in promise): Error: "CapcitorFlurry" plugin is not implemented on ios CapacitorException`

**Comment by jcesarmobile** [comment]
can you share your code?

**Reply by achakra21** [reply]
[demoplugin.zip](https://github.com/ionic-team/capacitor/files/6838987/demoplugin.zip)
here i have attached

**Reply by jcesarmobile** [reply]
I've tested the plugin in a new project and it works fine, can you provide a full app using the plugin?

**Comment by achakra21** [comment]
On Fri, 16 Jul 2021 at 7:03 PM, jcesarmobile ***@***.***>
wrote:

> can you share your code?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ionic-team/capacitor/discussions/4840#discussioncomment-1013283>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AALCHLOXOXLCWOXYSTA7JL3TYAYMFANCNFSM5APII6ZQ>
> .
>
Sure
